### ISSUE TITLE
rain map not working

### ISSUE THREAD

**Issue by carminemassarelli** [martykan/forecastie#253]
I use my api key but the rain map is not working.
I have forecastie 1.5 on lineageOS 14.1.
Is it a issue of the android version?
many thanks

**Comment by FridoDeluxe** [comment]
I can confirm this behavior. When trying to download the image tiles from the rain map I received HTTP response code 500, while temperature and wind were still working. Comparing the code of [map.html](https://github.com/martykan/forecastie/blob/a1e6359d327d863c662974cd9de6ac166f01954a/app/src/main/assets/map.html) the with OWM's [Weather map API](https://openweathermap.org/api/weathermaps) there are differences in the URLs.

I tested just the HTML page with a new URL and included the `appid`:

```javascript
// Rain
var rainLayer = new L.TileLayer("http://tile.openweathermap.org/map/precipitation_new/{z}/{x}/{y}.png?appid=" + getUrlParameter("appid"), {
```

This works for me. Hope this helps someone to replace all URLs and test if it also works inside the Android WebView (I can't do that at the moment).

**Comment by carminemassarelli** [comment]
I think it is not working this server link
http://{s}.maps.owm.io:8099/57b58d2a946d8a0100852aef/{z}/{x}/{y}?hash=d454ac88526eca77574f7d9c24a50efa

**Comment by robinpaulson** [comment]
@carminemassarelli @FridoDeluxe 
Cheers for working on this; would one of you be able to put together a pull request?

If not, is it as simple as replacing the lines as detailed in the comment above?

**Comment by carminemassarelli** [comment]
@robinpaulson @FridoDeluxe 
I have created a pull request in a fork because i do not have permission to write in the master.

**Comment by robinpaulson** [comment]
@carminemassarelli 
Marvellous. v1.6 will be out in a few days. As this patch is to fix broken functionality, I'll release v1.6.1 soon after, once we can see how v1.6 is going.
